src/main/java/org/jboss/logmanager/formatters/PatternFormatter.java

```diff
@@ -19,14 +19,20 @@
 
 package org.jboss.logmanager.formatters;
 
+import java.util.regex.Pattern;
+
 /**
  * A formatter which uses a text pattern to format messages.
  */
 public class PatternFormatter extends MultistepFormatter {
+    private static final Pattern CALLER_CHECK_PATTERN = Pattern.compile(".*(%M|%C|%F|%L|%l).*");
```

Comment:
Good point. We may just have to add something to the `FormatStep`. Since we're using Java 8 now though that shouldn't be an issue.